Explorer/Assets/DCL/Character/CharacterObject/Components/CharacterTargetPositionComponent.cs

```diff
@@ -0,0 +1,27 @@
+using UnityEngine;
+
+namespace DCL.Character.Components
+{
+    public struct CharacterTargetPositionComponent
+    {
+        public Vector3 LastPosition;
+        public Vector3 TargetPosition;
+        public Quaternion FinalRotation;
+        public bool IsPositionManagedByTween;
+        public bool IsRotationManagedByTween;
+
+        public CharacterTargetPositionComponent(
```

Comment:
I'm having doubts about the name and location of this class, what do you think of calling it something like `CharacterInterpolationMovementComponent` / `CharacterInterpolationComponent` ?

And maybe `/Character/CharacterMotion/Components/` is a better location for the file and namespace.